sound/soc/sof/topology.c

```diff
@@ -1896,6 +1896,23 @@ static int sof_complete(struct snd_soc_component *scomp)
 			}
 		}
 
+	/* load all 3rd party module libraries */
```

Comment:
As it is a loadable module, should we load it when necessary and unload it when stream is closed?